.travis.yml

```diff
@@ -17,7 +20,7 @@ addons: &clang6
 
 matrix:
   include:
-    - env: CXX="g++-7" CC="gcc-7"
+    - env: CXX="g++-7" CC="gcc-7" CPP_CHECK=1 USE_GLFW=1
```

Comment:
Only building hitting the GLFW code path with g++ right now -- most likely I'll just expand the build matrix to have more cases.